src/functions-and-arrays.js

```diff
@@ -26,13 +60,36 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(array) {
+if (array.length === 0) {
+  return null;
+}
+let sumOfNumbers = 0;
+for (let i= 0; i < array.length; i++) {
+  
+  sumOfNumbers += array[i];
+}
+return sumOfNumbers / array.length;
+}
+
 
 
 // Level 2: Array of strings
 const wordsArr = ['seat', 'correspond', 'linen', 'motif', 'hole', 'smell', 'smart', 'chaos', 'fuel', 'palace'];
 
-function averageWordLength() { }
+function averageWordLength(array) { 
+if (array.length === 0) {
+  return null;
+}
+let sumOfStrings = 0;
+for (let i = 0; i < array.length; i++) {
+  sumOfStrings += array[i];
```

Comment:
If you add that one you would get the tests passing, we want the average length of the word :) 
```suggestion
  sumOfStrings += array[i].length;
```